app/models.py

```diff
@@ -551,7 +551,7 @@ class SupplierDomain(db.Model):
 
     domain = relationship("Domain", back_populates="suppliers")
     supplier = relationship("Supplier", back_populates="domains")
-    assessments = relationship("Assessment")
+    assessments = relationship("Assessment", cascade="delete")
```

Comment:
does this mean there will be DB scripts to update the table?

Reply:
I believe `cascade` is a SQL Alchemy concept - not schema related